demos/fleet_provisioning/fleet_provisioning_csr/fleet_provisioning_config.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * AWS IoT Fleet Provisioning Client v1.0.0
```

Comment:
I would suggest not using the "Client" phrase for the library branding as it the library only exposes string utilities
```suggestion
 * AWS IoT Fleet Provisioning v1.0.0
```